src/rprblender/engine/viewport_engine.py

```diff
@@ -833,15 +833,11 @@ def sync_collection_instances(self, depsgraph, object_keys_to_export, material_o
         return res
 
     def remove_material_uvs_instances(self, mat, depsgraph):
-        """ Find and remove each unique instance of material by UVs combinations """
-        uvs = set(entry.data.rpr.uv_sets_names for entry in self.depsgraph_objects(depsgraph)
-                  if isinstance(entry, bpy.types.Mesh) and mat.name in entry.material_slots)
-
-        for entry in uvs:
-            for socket in ('Surface', 'Volume', 'Displacement'):
-                mat_key = material.key((mat.name, entry), input_socket_key=socket)
-                if mat_key in self.rpr_context.materials:
-                    self.rpr_context.remove_material(mat_key)
+        """ Find and remove each unique instance of material"""
```

Comment:
Seems remove_material_uvs_instances can be removed at all.

Reply:
I'm not sure what you mean by this comment.

Reply:
I mean, that this function can be removed at all, I checked: it isn't used anywhere now.